pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1973,3 +1990,48 @@ func hasTabletDevice(vmi *v1.VirtualMachineInstance) bool {
 	}
 	return false
 }
+
+func CalcDomDevice(ctx *ConverterContext, iface v1.VirtualMachineInstanceNetworkInterface) api.Interface {
+	return api.Interface{
+		Type: "ethernet",
+		Target: &api.InterfaceTarget{
+			Device:  generateTapDeviceName(iface.Name),
+			Managed: "no",
+		},
+		Model: &api.Model{Type: translateModel(ctx, "virtio")},
+		Alias: api.NewUserDefinedAlias("hotplug-" + iface.Name),
+	}
+}
+
+func generateTapDeviceName(networkName string) string {
+	const maxIfaceNameLen = 15
```

Comment:
I've done the test, it seems that maxIfaceNameLen =15 will cause plugPhase1 add `-nic` suffix cause rename fail ( ERANGE error ).
for example this generated interface name
`net49969d98044b` => `net49969d98044b-nic` cause `failed to switch pod interface with a dummy` error for bridge interface.

what do you think? @maiqueb